src/Uno.UWP/ApplicationModel/Contacts/ContactPhone.cs

```diff
@@ -0,0 +1,13 @@
+﻿
+namespace Windows.ApplicationModel.Contacts
+{
+	public partial class ContactPhone
+	{
+		public string Number { get; set; }
+		public ContactPhoneKind Kind { get; set; }
+		public ContactPhone()

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4900834)

Reply:
What is better? Adding empty constructor to implementation, or adding lines to diffignore?